client/app/components/project/package-section.hbs

```diff
@@ -0,0 +1,92 @@
+<div ...attributes>
+  <h2>{{@packageType}}</h2>
+
+  <div class="grid-x grid-padding-x">
+    <div class="cell large-6">
+      <ol class="no-bullet">
+        {{#each @packages as |package|}}
+          <Project::PackageListItem
+            @package={{package}}
+          />
+        {{/each}}
+      </ol>
+    </div>
+    <div class="cell large-6">
+      {{#let (flatten (map-by 'invoices' @packages)) as |invoices|}}
```

Comment:
Hmm why flatten? After map by, does(n't) it look like this? `[{ dcpGrandTotal: 323, dcpName: 'name'}, {dcpGrandTotal: 342, dcpName: 'name2'....]`